### ISSUE TITLE
Show all projects from an user

### ISSUE THREAD

**Issue by TrizML** [codecampleipzig/communic#49]
## Show all my projects
- As a **user**
- I want **see all the projects i've joined.**
- So that **I know how many of them I have.**

## Tasks
- [ ] Tests
- [ ] Documentation

**Comment by TrizML** [comment]
Due tu the fact that we are waiting on a DB we will postpone this story for the next sprint.

**Comment by TrizML** [comment]
The logic has been implemented for the mock data we are waiting on the DB.
